CMakeLists.txt

```diff
@@ -39,6 +40,11 @@ set(MMDEPLOY_TARGET_DEVICES "cpu" CACHE STRING "target devices to support")
 set(MMDEPLOY_TARGET_BACKENDS "" CACHE STRING "target inference engines to support")
 set(MMDEPLOY_CODEBASES "all" CACHE STRING "select OpenMMLab codebases")
 
+if ((NOT MMDEPLOY_BUILD_SDK_MONOLITHIC) AND MMDEPLOY_DYNAMIC_BACKEND)
```

Comment:
I suggest setting `MMDEPLOY_DYNAMIC_BACKEND` as `OFF` forcefully if `MMDEPLOY_BUILD_SDK_MONOLITHIC` is OFF